data/tlg0559/tlg0559.tlg016.xml

```diff
@@ -0,0 +1,1664 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader>
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Heronis Alexandrini geometricorum et stereometricorum reliquiae</title>
+<author xml:lang="lat">Heronis Alexandrini</author> 
+<editor xml:lang="lat">Fridericus Hultsch</editor>
+<sponsor>Harvard College Library</sponsor>
+ <funder>Harvard Library Arcadia Fund</funder>
+<principal>Gregory Crane</principal>
+<meeting/>
+<respStmt>
+ <persName xml:id="DDD">Digital Divide Data</persName>
+ <resp>Corrected and encoded the text</resp>
+</respStmt>
+<respStmt>
+ <persName>Rhea Lesage</persName>
+ <resp>Librarian for Hellenic Studies and Coordinator for the Classics
+Collection Development</resp>
+</respStmt>
+<respStmt>
+ <persName>Bruce Robertson</persName>
+ <resp>Technical Advisor</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+ <idno type="filename">tlg0559.tlg016</idno>
+<availability>
+ <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+</availability>
+<date>2017</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United State</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="lat">Heronis Alexandrini geometricorum et stereometricorum reliquiae</title>
+  <editor>
+   <persName>
+	 <name xml:lang="lat">Fridericus Hultsch</name>
+	</persName>
+  </editor>
+  <author xml:lang="lat" ref="urn:cts:greekLit:tlg0559.tlg016.opp-grc1">Heronis Alexandrini</author>
+  <imprint>
+	<publisher>Weidmann</publisher>
+	<pubPlace>Berlin</pubPlace>
+	<date>1864</date>
+  </imprint>
+  <biblScope unit="volume">1</biblScope>
+ </monogr>
+  <ref target="https://ia902701.us.archive.org/18/items/heronisalexandr00hultgoog/heronisalexandr00hultgoog.pdf">The Internet Archive</ref>
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+</encodingDesc>
```

Comment:
You need to add a `<refsDecl>` element with a `<cRefPattern>` element for each level of the citation scheme. See the Google Doc with instructions for how to do this.